ash/src/entry.rs

```diff
@@ -355,7 +365,7 @@ mod loaded {
 
     impl From<MissingEntryPoint> for LoadingError {
         fn from(err: MissingEntryPoint) -> Self {
-            LoadingError::MissingEntryPoint(err)
+            Self::MissingEntryPoint(err)
```

Comment:
Clippy woke up / started working all of a sudden. I don't understand why it's broken most of the time :/

Reply:
I think this wasn't getting coverage because the `loaded` feature wasn't enabled for clippy.

Reply:
https://github.com/MaikKlein/ash/pull/527